src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -146,6 +169,30 @@ const ChartsHolder = (props: {
                   </Disclaimer>
                 </>
               )}
+              <ChartHeader ref={contactTracingRef}>
+                {getMetricName(Metric.CONTACT_TRACING)}
+                <BetaTag>Beta</BetaTag>
+              </ChartHeader>
```

Comment:
something about this seemed to have changed the location of the beta tag?

Reply:
Yeah, this is an intended change based on moving towards new design: 
![image](https://user-images.githubusercontent.com/1658177/82093774-9ff0f280-96b0-11ea-9e34-3eee68489720.png)

https://www.figma.com/file/UUCr3ZpwaPV4o1KuGdSCNl/Covid?node-id=3985%3A3471